docs/Gemfile

```diff
@@ -0,0 +1,7 @@
+source 'https://rubygems.org'
```

Comment:
Commit the related `Gemfile.lock` to make the build more reproducible.

Reply:
Done